EIPS/eip-3475.md

```diff
@@ -1,18 +1,22 @@
 ---
 eip: 3475
-title: Multiple Callable Bonds
-author: Yohji Sakamoto (@sgmfinance)
-discussions-to: https://github.com/ethereum/EIPs/issues/3467
+title: ERC-3475 Multiple Callable Bonds Standard
+author: Yu LIU <info@debond.org>
+discussions-to: https://github.com/ethereum/EIPs/pull/4858/
 type: Standards Track
 category: ERC
-status: Draft
+status: Review
 created: 2021-04-05
+last changed: 2022-02-28
+
 ---
 
 ## Simple Summary
 
 A standard interface for contract, that manage multiple callable bonds. A single contract includes any given number of bond classes, bond nonce, bond balance of an address. This standard provides independent functions to read, transfer any collection of bonds, as well as allow bonds to be redeemed from the bond issuer if certain conditions are met. This token standard can replace current ERC20 LP token. ERC-3475 has more complex data structure, which will allow the LP token to store more information, and allow the developer to build more sophisticated logic for the redemption and reward system of the DEFI project in question.
 
+
+
```

Comment:
```suggestion
```
Simple Summary is no longer part of EIPs, `description` in the header should be used instead (and it should be short).